host-ext/nacl_src/kiwi.cc

```diff
@@ -246,6 +246,12 @@ class KiwiInstance : public pp::Instance {
         return false;
     }
 
+    /* Recieves and processes initialization information */
+    void SocketParseInitInformation(const char* data, int datalen) {
```

Comment:
Is the idea that later revs would just make datalen longer, so you don't need to bother with versioning?